src/ods-hyperlink.js

```diff
@@ -147,7 +149,7 @@ class OdsHyperlink extends LitElement {
         ?download="${this.download}"
         role="${ifDefined(this.role === 'button' || this.role === 'tab' ? this.role : undefined)}"
         rel="${ifDefined(this.target || this.rel ? this.getReltype(this.target, this.rel) : undefined)}"
-        class="${this.getAnchortype(this.role, this.href)} ${this.getContext(this.inline)} ${this.getTheme(this.darktheme)} ${this.getCta(this.cta)} ${this.getTabState(this.tabisactive)}"
+        class="${this.getStyles(this.role, this.darktheme, this.inline, this.cta, this.buttontype)} ${this.getTabState(this.tabisactive)}"
```

Comment:
Do you need to pass these arguments here if you're accessing them all with `this` in `getStyles`?

```suggestion
        class="${this.getStyles()} ${this.getTabState(this.tabisactive)}"
```

Reply:
Updated ... thanks 👍